seatunnel-connectors-v2/connector-activemq/src/main/java/org/apache/seatunnel/connectors/seatunnel/activemq/sink/ActivemqSinkFactory.java

```diff
@@ -72,9 +66,4 @@ public OptionRule optionRule() {
                         WARN_ABOUT_UNSTARTED_CONNECTION_TIMEOUT)
                 .build();
     }
-
-    @Override
-    public TableSink createSink(TableSinkFactoryContext context) {
-        return () -> new ActivemqSink(context.getOptions(), context.getCatalogTable());
-    }
```

Comment:
Please use `createSink` to create new instance. Do not use SPI in https://github.com/apache/seatunnel/pull/8581/files#diff-5e51ad244c8ce82135d5edbc4f9a925492b1e479c25f45f8ae722c36bd09edebR45